test/URMCompoundManager.invariant.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.30;
+
+// External Imports
+import {Test} from "forge-std/Test.sol";
+
+// Internal Imports
+import {URMCompoundManager} from "src/contracts/urm/URMCompoundManager.sol";
+import {MockCompoundTimelock} from "test/mocks/MockCompoundTimelock.sol";
+import {FakeProtocolContract} from "test/fakes/FakeProtocolContract.sol";
+import {URMCompoundManagerHandler} from "test/handlers/URMCompoundManager.handler.sol";
+import {ICompoundTimelockTarget} from "src/interfaces/ICompoundTimelockTarget.sol";
+
+contract URMCompoundManagerInvariantTest is Test {
+  URMCompoundManager public urm;
+  MockCompoundTimelock public timelockTarget;
+  FakeProtocolContract[] public targets;
+  URMCompoundManagerHandler public handler;
+
+  address public admin = makeAddr("admin");
+  address public guardian = makeAddr("guardian");
+  uint256 public delay = 5 seconds;
+
+  function setUp() public {
+    timelockTarget = new MockCompoundTimelock();
+    timelockTarget.setDelay(delay);
+
+    targets = new FakeProtocolContract[](3);
+    targets[0] = new FakeProtocolContract(admin);
+    targets[1] = new FakeProtocolContract(admin);
+    targets[2] = new FakeProtocolContract(admin);
+
+    urm = new URMCompoundManager(address(timelockTarget), admin, guardian, delay, delay);
+
+    handler = new URMCompoundManagerHandler(urm, admin, guardian, targets);
+
+    // target the handler for invariant testing
+    targetContract(address(handler));
+  }
+
+  function invariant_rollbackWithSameIdCannotBeProposedMoreThanOnce() public view {}
+
+  function invariant_expiredRollbackIsNotExecutable() public view {}
+
+  function invariant_expiredRollbackIsNotCancelable() public view {}
+
+  function invariant_executedRollbackIsNotCancelable() public view {}
+
+  function invariant_executedRollbackIsNotExecutable() public view {}
+
+  function invariant_cancelledRollbackIsNotCancelable() public view {}
+
+  function invariant_cancelledRollbackIsNotExecutable() public view {}
+
+  function invariant_executedRollbackUpdatesStateOfTarget() public view {}
+
+  function invariant_cannotExecuteRollbackWithoutQueueing() public view {}
+
+  function invariant_cannotCancelRollbackWithoutQueueing() public view {}
```

Comment:
^ @wildmolasses would love for you to just look at these and tell me if there are good invariants